lib/databases/timescale/lootboxes/rewards.go

```diff
@@ -0,0 +1,141 @@
+// Copyright 2023 Fluidity Money. All rights reserved. Use of this
+// source code is governed by a GPL-style license that can be found in the
+// LICENSE.md file.
+
+package lootboxes
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/fluidity-money/fluidity-app/lib/log"
+	"github.com/fluidity-money/fluidity-app/lib/timescale"
+)
+
+// UserLootboxCount to group an address and their respective lootboxes
+type UserLootboxCount struct {
+	Address string `json:"address"`
+	LootboxCount float64 `json:"lootbox_count"`
+}
+
+// InsertTopUserReward to insert lootboxes for the given users for their activity during the airdrop.
+// Expects 5 users to reward
+func InsertTopUserReward(currentTime time.Time, users []UserLootboxCount) {
+    if len(users) != 5 {
+        log.Fatal(func(k *log.Log) {
```

Comment:
Here, could go `if lenUsers := len(users); lenUsers != 5`, also, make the 5 a constant